packages/dialog/test/draggable-resizable.test.js

```diff
@@ -68,6 +68,7 @@ describe('helper methods', () => {
     dialog2.renderer = (root) => {
       root.innerHTML = '<div>Modeless dialog 2</div>';
     };
+    await nextFrame();
```

Comment:
We discussed the same thing with @vursen and added [`nextUpdate()` helper](https://github.com/vaadin/testing-helpers/pull/9) to handle Lit based components updates instead of `nextFrame()`, which is indeed confusing. I will update this PR to use the new helper.

Reply:
Changed to use `nextUpdate()` where dialog properties are set, except for `opened` which requires us waiting for `nextRender()` in order to ensure that `aria-hidden` setting logic is set at the right timing.